usr/share/rear/restore/SUSE_LINUX/91_create_missing_directories.sh

```diff
@@ -1,7 +1,23 @@
 #
-# SuSE likes to have stuff under /media
-#
-# create missing directories
-pushd /mnt/local >&8
-mkdir -p media/cdrom media/floppy
+# In some cases SUSE has /media plus /media/cdrom and /media/floppy.
+# On SLE11 the filesystem RPM contains them.
+# On SLE12 they are dropped from the filesystem RPM because nowadays /run/media is used.
+# There is the SUSE-internal issue https://bugzilla.suse.com/show_bug.cgi?id=890198
+# and the "SUSE Linux Enterprise Server 12 Release Notes" mention it
+# see https://www.suse.com/releasenotes/x86_64/SUSE-SLES/12/ that reads:
+#   "/run/media/<user_name> is now used as top directory for removable
+#    media mount points. It replaces /media , which is not longer available."
+# Therefore for SLE12 "rear recover" must no longer create them.
+# FIXME: The following test for SLE12 products is intentionally sloppy
+# because I <jsmeix@suse.de> have no better idea how to test for various
+# posible SLE12-based products like "SUSE Linux Enterprise Server 12"
+# "SUSE Linux Enterprise Desktop 12" "SUSE Linux Enterprise Server 12 SP1"
+# "SUSE Linux Enterprise Desktop 12 SP1" "SUSE Linux Enterprise <whatever> <whichever>".
```

Comment:
I found out what there is on openSUSE 13.1 and 13.2:

Both have /etc/os-release and
both do not have media/cdrom and media/floppy

Therefore current SUSE_LINUX/91_create_missing_directories.sh
works for SLE11, openSUSE 13.1 13.2 and SLE12 and then also for openSUSE Leap 42.1 because that is based on SLE12.